content/applications/general/email_communication/email_servers.rst

```diff
@@ -5,122 +5,122 @@ Send and Receive Emails in Odoo with an Email Server
-a message, Odoo will try to redistribute that same email to other subscribers
-in the thread. But if the domain *customer.example.com* forbids that kind of
-usage for security, the Odoo's redistributed email would get
-rejected by some recipients' mail servers.
+For example, if a customer with address *mary\@customer.example.com* responds to a message, Odoo
+will try to redistribute that same email to other subscribers in the thread. But if the domain
+*customer.example.com* forbids that kind of usage for security, the Odoo's redistributed email
+would get rejected by some recipients' mail servers.
 
-To avoid those kind of problems, you should make sure all emails use a "From"
-address from your authorized domain.
+To avoid those kind of problems, you should make sure all emails use a "From" address from your
+authorized domain.
 
-If your MTA supports `SRS (Sender Rewriting Scheme)
-<https://en.wikipedia.org/wiki/Sender_Rewriting_Scheme>`_, you can enable it
-to handle these situations. However, that is more complex and requires more
-technical knowledge that is not meant to be covered by this documentation.
+If your MTA supports `SRS (Sender Rewriting Scheme) <https://en.wikipedia.org/wiki/Sender_Rewriting
+_Scheme>`_, you can enable it to handle these situations. However, that is more complex and
+requires more technical knowledge that is not meant to be covered by this documentation.
 
 Instead, you can also configure Odoo to do something similar by itself:
 
-#.  Set your domain name in the General Settings.
+#. Set your domain name in the :guilabel:`General Settings`.
 
-    .. image:: email_servers/alias-domain.png
-       :align: center
+   .. image:: email_servers/alias-domain.png
+      :align: center
+      :alt: Setting the domain alias configuration on Odoo.
 
-#.  In developer mode, go to :menuselection:`Settings --> Technical -->
-    Parameters --> System Parameters`.
+#. In developer mode, go to :menuselection:`Settings --> Technical --> Parameters --> System
+   Parameters`.
 
-#.  Add one system parameter from these:
+#. Add one system parameter from these:
 
-    * If you want *all* your outgoing messages to use the same "From" address,
-      use the key ``mail.force.smtp.from`` and set that address as value
-      (such as ``outgoing@mycompany.example.com``).
+   * If you want *all* your outgoing messages to use the same "From" address, use the key
+     `mail.force.smtp.from` and set that address as value (such as `outgoing@mycompany.example.com`
+     ).
 
-    * If you want to keep the original "From" address for emails that use your
-      same domain, but change it for emails that use a different domain, use
-      the key ``mail.dynamic.smtp.from`` and set as value the email address
-      that should be used in those cases (such as
-      ``outgoing@mycompany.example.com``).
+   * If you want to keep the original "From" address for emails that use your same domain, but change
```

Comment:
Replace `If you want to keep the original "From" address for emails that use your same domain` with `To keep the original "From" address for emails that use the same domain`